architecture/topics/aws_getting_started_install.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * admin_guide/aws-getting-started.adoc
+
+[id='aws-getting-started-install-{context}']
+= Installing the AWS Broker into an existing {product-title} cluster
+
+To install the AWS broker into your {product-title} cluster:
+
+. Create a directory for the AWS Broker installation files:
++
+----
+mkdir awssb
+cd awssb
+----
+
+. Download the installation files:
++
+----
+wget https://raw.githubusercontent.com/awslabs/aws-servicebroker/release-v1.0.0-beta.2/packaging/openshift/deploy.sh
+wget https://raw.githubusercontent.com/awslabs/aws-servicebroker/release-v1.0.0-beta.2/packaging/openshift/aws-servicebroker.yaml
+wget https://raw.githubusercontent.com/awslabs/aws-servicebroker/release-v1.0.0-beta.2/packaging/openshift/parameters.env
+chmod +x deploy.sh
+----
```

Comment:
We need a call out that issues with these scripts should be biased with AWS.

Reply:
Added a note to this effect.